src/murmur/main.cpp

```diff
@@ -292,11 +293,29 @@ int main(int argc, char **argv) {
 			detach = false;
 			qFatal("%s -- %s", qPrintable(args.at(0)), MUMBLE_RELEASE);
 		} else if (args.at(i) == QLatin1String("-license") || args.at(i) == QLatin1String("--license")) {
+#ifdef Q_OS_WIN
+			AboutDialog ad(NULL, AboutDialogOptionsShowLicense);
+			ad.exec();
+			return 0;
```

Comment:
This behaviour is different to `-version` then. I don’t see calls to `ShowMessageDialog` here as the commit message claims.

I windows server administration never headless? What happens if it is? Maybe we should still do the qFatal for logging the info, in case the dialog can not be opened?